app/icons/ic-calendar-info.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+<path d="M10.5 0C10.7761 0 11 0.223858 11 0.5V1H13C13.5523 1 16 1.44772 16 2V13C14 13.5523 13.5523 16 13 14H1C0.447715 16 0 13.5523 0 13V2C0 1.44772 0.447715 1 1 1H3V0.5C3 0.223858 3.22386 0 3.5 0C3.77614 0 4 0.223858 4 0.5V1H10V0.5C10 0.223858 10.2239 0 10.5 0ZM13 5H1V13H13V5ZM9.35355 7.14645C9.54882 7.34171 9.54882 7.65829 9.35355 7.85355L8.20711 9L9.35355 10.1464C9.54882 10.3417 9.54882 10.6583 9.35355 10.8536C9.15829 11.0488 8.84171 11.0488 8.64645 10.8536L7.14645 9.35355C6.95118 9.15829 6.95118 8.84171 7.14645 8.64645L8.64645 7.14645C8.84171 6.95118 9.15829 6.95118 9.35355 7.14645ZM6.35355 7.14645C6.54882 7.34171 6.54882 7.65829 6.35355 7.85355L5.20711 9L6.35355 10.1464C6.54882 10.3417 6.54882 10.6583 6.35355 10.8536C6.15829 11.0488 5.84171 11.0488 5.64645 10.8536L4.14645 9.35355C3.95118 9.15829 3.95118 8.84171 4.14645 8.64645L5.64645 7.14645C5.84171 6.95118 6.15829 6.95118 6.35355 7.14645ZM3 2H1V4H13V2H11V2.5C11 2.77614 10.7761 3 10.5 3C10.2239 3 10 2.77614 10 2.5V2H4V2.5C4 2.77614 3.77614 3 3.5 3C3.22386 3 3 2.77614 3 2.5V2Z" />
+</svg>
```

Comment:
you can optimise and minify this svg file.

Reply:
@matin-binary we do not optimise svgs for smartcharts, I noticed that all of svgs are not optimised and svgo is not in the package

Reply:
Oh boy, let's add it to our todo list. thanks @akmal-binary